mlf_core/sync/sync.py

```diff
@@ -61,8 +66,8 @@ def __init__(self, project_dir,
         self.patch_update = patch_update
         self.gh_username = gh_username if gh_username else load_github_username()
         self.token = token if token else decrypt_pat()
-        self.repo_owner = repo_owner if repo_owner else gh_username
         self.dot_mlf_core = {}
+        self.repo_owner = self.gh_username
```

Comment:
Ah, now I see.
Actually I am pretty sure that I ran into some edge case with qube when doing exactly that.

Can you verify that it works for organization repositories?

Reply:
I did a test with cookietemple and it worked

Reply:
If only I remember which edge case I have in the back of my mind ahhhhhhhhhhh